source/views/sis/course-search/detail/index.js

```diff
@@ -82,28 +97,38 @@ function Information({course}: {course: CourseType}) {
 }
 
 function Schedule({course}: {course: CourseType}) {
-	if (!course.times) {
+	if (!course.offerings) {
 		return null
 	}
-	const times = convertTimeStringsToOfferings(
-		{
-			times: course.times,
-			locations: course.locations,
-		},
-		{groupBy: 'day'},
-	)
-	const schedule = times.map(time => {
-		const hours = time.times.map(time => {
-			const start = moment.tz(time.start, 'hmm', CENTRAL_TZ).format('h:mm A')
-			const end = moment.tz(time.end, 'hmm', CENTRAL_TZ).format('h:mm A')
+	const groupedByDay = groupBy(course.offerings, 'day')
+	const schedule = map(groupedByDay, (offerings, day) => {
+		const timesFormatted = offerings.map(offering => {
+			const start = moment
+				.tz(offering.start, 'H:mm', CENTRAL_TZ)
+				.format('h:mm A')
+			const end = moment.tz(offering.end, 'H:mm', CENTRAL_TZ).format('h:mm A')
 			return `${start} – ${end}`
 		})
+		const locations = offerings.map(offering => offering.location)
+
+		const timelocs = zip(timesFormatted, locations)
+
+		const timelocsObj = timelocs.map(timelocs =>
+			zipObject(['time', 'location'], timelocs),
+		)
+
+		const rightDetail = timelocsObj.map(timeloc => (
+			<Text key={timeloc.time} style={styles.rightDetail}>
+				<Text style={styles.time}>{timeloc.time} </Text>
+				<Text style={styles.location}>{`(${timeloc.location})`}</Text>
```

Comment:
Just change this to `>({timeloc.location})</Text>` – we don't need the extra string creation within JSX.